src/titiler/core/pyproject.toml

```diff
@@ -35,7 +35,7 @@ dependencies = [
     "numpy",
     "pydantic~=2.0",
     "rasterio",
-    "rio-tiler>=6.0,<7.0",
+    "rio-tiler@https://github.com/SenteraLLC/rio-tiler/archive/refs/heads/6.1.0-sentera-changes-dev.tar.gz",
```

Comment:
🚨 Commit after merging SenteraLLC/rio-tiler#3 and creating a `6.1.0-0` release 🚨 
```suggestion
    "rio-tiler@https://github.com/SenteraLLC/rio-tiler/archive/refs/heads/6.1.0-0.tar.gz",
```